packages/components/src/utils/backgrounds.ts

```diff
@@ -0,0 +1,108 @@
+/**
+ * External dependencies
+ */
+import { css, SerializedStyles } from '@emotion/react';
+import colorize from 'tinycolor2';
+
+/**
+ * Internal dependencies
+ */
+import COLORS from './colors-values';
+
+export const SUPPORTED_COLORS = [
+	'blue',
+	'red',
+	'purple',
+	'green',
+	'yellow',
+	'orange',
+	'darkGray',
+	'lightGray',
+] as const;
+
+export type SupportedColors = typeof SUPPORTED_COLORS[ number ];
+
+function assertIsSupportedColor(
+	color: any
+): asserts color is SupportedColors {
+	if ( ! SUPPORTED_COLORS.includes( color ) ) {
+		throw new Error(
+			`'color' must be one of the supported colors: ${ SUPPORTED_COLORS.join(
+				', '
+			) }`
+		);
+	}
+}
+
+export function getBackgroundColor(
+	color: SupportedColors,
+	options: { isBold?: boolean } = {}
+): SerializedStyles {
+	assertIsSupportedColor( color );
+
+	const { isBold } = options;
+
+	const colorized = colorize( color );
+
+	switch ( color ) {
+		case 'purple':
+		case 'green':
+			colorized.lighten( 60 );
+			colorized.desaturate( 60 );
+			break;
+		case 'blue':
+		case 'red':
+		case 'orange':
+			colorized.lighten( 40 );
+			colorized.desaturate( 10 );
+			break;
+		case 'darkGray':
+			colorized.lighten( 25 );
+			break;
+		case 'lightGray':
+			colorized.lighten( 13 );
+			break;
+		case 'yellow':
+			colorized.lighten( 40 );
+			break;
+	}
+
+	if ( isBold ) {
+		colorized.darken( 20 );
+	}
+
+	// if ( isBold ) {
+	// 	colorized.lighten( 30 );
+	// } else if ( isSubtle ) {
+	// 	colorized.lighten( 90 );
+	// }
```

Comment:
Can this code be removed? Is the `isSubtle` option still a requirement, or can we do without it?

Reply:
isSubtle is complete unused so I removed it. The comments left were a mistake. Removed in the next commit